contracts/modules/cover/Cover.sol

```diff
@@ -467,7 +471,7 @@ contract Cover is ICover, MasterAwareV2, IStakingPoolBeacon, ReentrancyGuard, Mu
 
       if (remainder > 0) {
         // solhint-disable-next-line avoid-low-level-calls
-        (bool ok, /* data */) = address(msg.sender).call{value: remainder}("");
+        (bool ok, /* data */) = address(buyer).call{value: remainder}("");
```

Comment:
this might be undesirable since the buyer pays with WETH, but let's leave it as is and I'll get to it when I rebase on top of this. It's going to create conflicts anyway.

Reply:
would help to leave a todo though!